docs/mutations.md

```diff
@@ -159,3 +159,56 @@ Do you have a suggestion for a (new) mutator? Feel free to create an [issue](htt
 
 ## Regular Expressions (_regex_)
 For the full list of all available regex mutations, see the [regex mutator docs](./regex-mutations.md).
+
+## Math Methods (_math_)
+|      Original           |         Mutated         |
+| ----------------------- | ----------------------- |
+| Acos()                  | Acosh()                 |
+| Acos()                  | Asin()                  |
+| Acos()                  | Atan()                  |
```

Comment:
Is there a specific reason to have multiple mutated variants for the same thing? 

I vaguely remember from my trigonometry lessons at school 15 years ago that there are some cases when some of the pairs will have the same values (like errrrhhhhh sin and cos for 45 degrees?) but I think philosophically Stryker in the vast majority of cases puts up with that (@rouke-broersma correct me if I am wrong).

Like, Stryker changes + to - and other way round, but it doesn't also change + to * or /, even though `x + 0` and `x - 0` will give the same result. AFAIU the framework kind of tries to find a balance between the amount of value and amount of generated mutants (because the amount eventually affects the performance).

TL;DR - I'd stick with one mutant for each expression.

Reply:
Hello.

The case with `sin(x) = cos(x)` was discussed in the initial issue - they are indeed equal for 45 and 225 degrees. From my experience with mutation testing, if someone gets such survived mutant then this is a good point to improve tests - just add a new test case with an another angle.

The same for other swaps, they will test well other "magic" angles that can be used in incomplete tests - e.g. `sin(x) = sinh(x)` for 0 and `sin(x) = tan(x)` for `pi*n`.

And swap trigonometric <-> hyperbolic function looks like a real possible error with the function selection in IntelliSense :D

Reply:
Alright. I mean, I haven't worked in codebases where this stuff was heavily used, just wanted to share the concern that it can generate many mutants. But yeah, that's an advanced mutator anyway and will probably not be applicable for most repos.